src/components/input/Input.jsx

```diff
@@ -0,0 +1,35 @@
+import styles from "./Input.module.css";
+
+const Input = ({
+  placeholder = "placeholder",
+  disabled = false,
+  error = false,
+  errorMessage = "",
+  value,
+  onChange,
+}) => {
+  const inputClassName = [
+    styles.input,
+    error && styles.error,
+    disabled && styles.disabled,
+  ]
+    .filter(Boolean)
+    .join(" ");
+
+  return (
+    <div className="styles.inputContainer">
+      <input
+        className={inputClassName}
```

Comment:
`clsx` 라이브러리를 활용하면 더 직관적으로 조건부 클래스 적용이 가능해요.

```suggestion
import clsx from 'clsx';
...
  return (
    <div className="styles.inputContainer">
      <input
        className={clsx(styles.input, {
          [styles.error]: error,
          [styles.disabled]: disabled,
        }}
```